sdk/network/Azure.ResourceManager.Network/samples/Generated/Samples/Sample_P2SVpnGatewayResource.cs

```diff
@@ -241,7 +241,7 @@ public async Task GetP2SVpnConnectionHealthDetailed_P2SVpnGatewayGetConnectionHe
         public async Task DisconnectP2SVpnConnections_DisconnectVpnConnectionsFromP2sVpnGateway()
         {
             // Generated from example definition: specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/examples/P2sVpnGatewaysDisconnectP2sVpnConnections.json
-            // this example is just showing the usage of "P2sVpnGateways_DisconnectP2SVpnConnections" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "P2sVpnGateways_DisconnectP2sVpnConnections" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `P2s`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/virtualWan.json#L4685